tastypie/contrib/gis/resources.py

```diff
@@ -75,3 +76,9 @@ def filter_value_to_python(self, value, field_name, filters, filter_expr,
             except ValueError:
                 pass
         return value
+
+
+class ModelResource(GeometryModelMixin, resources.ModelResource):
```

Comment:
I've generally found that when you subclass this further with addition mixins, having the Mixin first can create MRO issues.  I don't have a concrete example of this, however.